src/main/java/com/wavefront/sdk/jersey/WavefrontJerseyFilter.java

```diff
@@ -317,6 +322,13 @@ public void filter(ContainerRequestContext containerRequestContext,
         }
         wfJerseyReporter.incrementDeltaCounter(new MetricName(
             "response.errors.aggregated_per_application", overallAggregatedPerApplicationMap));
+        Optional<Pair<String, String>> ApiPathOnlyOptional =
+            MetricNameUtils.metricNameAndPath(request, "response.");
```

Comment:
why do this when you already have 
 String responseMetricKey = responseOptionalPair.get(); above ?

Reply:
responseMetricKey has status code, while this metric doesn't need status code. Will modify original call to avoid extra call.